x-pack/plugins/ml/server/routes/notifications.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { NotificationsService } from '../models/notifications_service';
+import {
+  getNotificationsCountQuerySchema,
+  getNotificationsQuerySchema,
+} from './schemas/notifications_schema';
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../types';
+
+export function notificationsRoutes({ router, routeGuard }: RouteInitialization) {
+  router.get(
+    {
+      path: '/api/ml/notifications',
+      validate: {
+        query: getNotificationsQuerySchema,
+      },
+      options: {
+        tags: ['access:ml:canGetAnnotations'],
```

Comment:
@jgowdyelastic not sure about the tags, could you please assist with that?

Reply:
I think these could be `['access:ml:canGetJobs', 'access:ml:canGetDataFrameAnalytics', 'access:ml:canGetTrainedModels']`

Reply:
updated in https://github.com/elastic/kibana/pull/140613/commits/1f872cf1fc4029094523f32772f9c033c11689b9